microsoft-edge/toc.yml

```
@@ -1160,87 +1160,87 @@
 # PWA
     - name: Progressive Web Apps
       items:
-        - name: Overview of Progressive Web Apps (PWAs)
-          href: progressive-web-apps-chromium/index.md
+        - name: Progressive Web Apps (PWAs) documentation
+          href: ./progressive-web-apps-chromium/landing/index.yml
+          displayName: landing
 
-        - name: Get started with PWAs
-          href: progressive-web-apps-chromium/how-to/index.md
-          displayName: Get started with Progressive Web Apps  # top-of-page title
+        - name: Overview and getting started
+          items:
+            - name: Overview of Progressive Web Apps (PWAs)
+              href: ./progressive-web-apps-chromium/index.md
 
-        - name: Best practices for PWAs
-          href: progressive-web-apps-chromium/how-to/best-practices.md
+            - name: What's New in PWAs
+              href: ./progressive-web-apps-chromium/whats-new/pwa.md
+              displayName: release notes, announcements
+
+            - name: Get started with PWAs
+              href: ./progressive-web-apps-chromium/how-to/index.md
 
-        - name: How to
+            - name: Use PWAs in Microsoft Edge
+              href: ./progressive-web-apps-chromium/ux.md
+
+        - name: Integrate a PWA into Windows
           items:
-            - name: Publish a Progressive Web App to the Microsoft Store
-              href: progressive-web-apps-chromium/how-to/microsoft-store.md
+            - name: Display content in the title bar area using Window Controls Overlay  # todo: add "Share dialog" to title
+              href: ./progressive-web-apps-chromium/how-to/window-controls-overlay.md
 
-            - name: Experimental features and origin trials for PWAs
-              href: progressive-web-apps-chromium/how-to/origin-trials.md
+            - name: Share content with other apps  # todo: add "jump lists" to title
+              href: ./progressive-web-apps-chromium/how-to/share.md
 
-            - name: Use Progressive Web Apps in Microsoft Edge
-              href: progressive-web-apps-chromium/ux.md
+            - name: Define app shortcuts (long-press or right-click menus)
+              href: ./progressive-web-apps-chromium/how-to/shortcuts.md
 
-            - name: Debug Progressive Web Apps
-              href: progressive-web-apps-chromium/how-to/debug.md
+            - name: Display a PWA widget in the Windows Widgets Board
+              href: ./progressive-web-apps-chromium/how-to/widgets.md
 
-            - name: Use a web app manifest to integrate a PWA into the OS
-              href: progressive-web-apps-chromium/how-to/web-app-manifests.md
+            - name: Build a PWA for the sidebar in Microsoft Edge
+              href: ./progressive-web-apps-chromium/how-to/sidebar.md
 
-            - name: Use a service worker to manage network requests
-              href: progressive-web-apps-chromium/how-to/service-workers.md
+        - name: Publish and integrate with the Microsoft Store
```

```suggestion
        - name: Publish and integrate with the Microsoft Store
```
alts:
        - name: Publish and integrate with the Microsoft Store
        - name: Publish PWAs with the Microsoft Store
        - name: Publish a PWA with the Microsoft Store
        - name: Microsoft Store publishing and integration
        - name: Publish and integrate with the Microsoft Store
        - name: Microsoft Store: Publishing and integrating a PWA